src/OSPSuite.Infrastructure.Import/Core/Exceptions/ParseErrorDescription.cs

```diff
@@ -0,0 +1,66 @@
+﻿using OSPSuite.Assets;
+using System.Collections.Generic;
+
+namespace OSPSuite.Infrastructure.Import.Core.Exceptions
+{
+   public abstract class ParseErrorDescription
+   {
+      public string Message { get; protected set; }
+   }
+
+   public class InvalidDimensionParseErrorDescription : ParseErrorDescription
+   {
+      public InvalidDimensionParseErrorDescription(string invalidUnit, string mappingName)
+      {
+         Message = Error.InvalidDimensionException(invalidUnit, mappingName);
+      }
+   }
+
+   public class NaNParseErrorDescription : ParseErrorDescription
+   {
+      public NaNParseErrorDescription()
+      {
+         Message = Error.NaNOnData;
+      }
+   }
+
+   public class MismatchingArrayLengthsParseErrorDescription : ParseErrorDescription
```

Comment:
Why do we have all those classes instead of using the string? associated with the message?